pype/hooks/global/pre_with_windows_shell.py

```diff
@@ -11,7 +11,7 @@ class LaunchWithWindowsShell(PreLaunchHook):
     """
 
     order = 10
-    app_groups = ["nuke", "nukex", "hiero", "nukestudio"]
+    app_groups = ["resolve", "nuke", "nukex", "hiero", "nukestudio"]
```

Comment:
Is this required?

Reply:
Not sure if I need it but as I see `hiero` and `nukestudio` here I was presuming even `resolve` is good to go here.

Reply:
It is because Nuke requires opened command line. Please remove if launching of resolve works without.